src/vm/ecalllist.h

```diff
@@ -690,6 +689,7 @@ FCFuncStart(gRuntimeThreadFuncs)
     FCFuncElement("InterruptInternal", ThreadNative::Interrupt)
     FCFuncElement("JoinInternal", ThreadNative::Join)
     QCFuncElement("GetOptimalMaxSpinWaitsPerSpinIterationInternal", ThreadNative::GetOptimalMaxSpinWaitsPerSpinIteration)
+    FCFuncElement("GetCurrentProcessorNumber", ThreadNative::GetCurrentProcessorNumber)
```

Comment:
rename to GetCurrentProcessorNumberInternal?

Reply:
FCall naming conventions say to use the Internal prefix only if necessary. Internal is not necessary here. https://github.com/dotnet/coreclr/blob/master/Documentation/botr/mscorlib.md#naming-convention.